### DISCUSSION TITLE
Renovate keeps reverting lockfileversion to 1 despite engine constraint

### DISCUSSION THREAD

**Discussion by mieliespoor** [renovatebot/renovate#18044] [Request Help] [answered]
Having read discussion #13027 my understanding was that this should be resolved.

We are running Renovate v32.159.5 and still seeing lockfileversion being downgraded to 1.

Should we be setting `binarySource=docker` what is the fix here?

**Comment by mieliespoor** [comment, marked as answer]
I just deployed 32.208.2 now and also specifying `RENOVATE_BINARY_SOURCE=install` and still seeing lockfileversion being downgraded to 1.

**Reply by rarkins** [reply]
Please do this:
- Find an existing PR which is wrong
- Tick the rebase/retry checkbox to get Renovate to retry it
- Copy/paste the debug log *for that branch* here

**Reply by mieliespoor** [reply]
I had a look at logs now of package updates that ran after the upgrade and introduction of the ENV variable. These new pull-requests didn't downgrade the lockfileversion. I marked the pr's before the upgrade for rebase and will see what happens there, but I think this is actually solved. 

I will review the rebased pr's in the morning and then possibly closing this as resolved. Thank you for the hint.
